apps/studio/components/ui/ProductMenu/ProductMenuItem.tsx

```diff
@@ -35,9 +35,7 @@ const ProductMenuItem = ({
           className={'flex items-center gap-2 truncate w-full ' + textClassName}
         >
           <span className="truncate">{name} </span>
-          {label !== undefined && (
-            <span className="text-orange-800 text-xs font-normal truncate">{label}</span>
-          )}
+          {label !== undefined && <Badge color="yellow">{label}</Badge>}
```

Comment:
```suggestion
          {label !== undefined && <Badge color="amber">{label}</Badge>}
```

Reply:
@MildTomato - Just pushed with `amber` and looks like this:
<img width="167" alt="Screenshot 2024-02-27 at 14 16 36" src="https://github.com/supabase/supabase/assets/22655069/5b629402-cb25-4189-ad08-d25bf72cbc83">